ci-operator/step-registry/netobserv/perf-test/post-workload/OWNERS

```diff
@@ -0,0 +1,14 @@
+approvers:
+- memodi
+- jotak
+- jpinsonneau
+- oliviercazade
```

Comment:
not sure if this is case sensitive or not, but Olivier's username has capitals: https://github.com/OlivierCazade

Reply:
thanks, I updated it everywhere.